server/@types/express/index.d.ts

```diff
@@ -11,6 +11,9 @@ declare module 'express-session' {
     offenderNo: string
     visitorList: VisitorListItem[]
     adultVisitors: VisitorListItem[]
+    slotsList: VisitSlotList
```

Comment:
Added these to session so list of slots and filter settings can be saved on GET request to be re-rendered in POST route should validation fail. For `timeOfDay` and `dayOfWeek`, I did consider just keeping these in the URL params but that would have meant duplication of the validation....